src/compiler/scala/tools/nsc/typechecker/Implicits.scala

```diff
@@ -968,6 +1003,13 @@ trait Implicits {
        *   - find the most likely one
        *   - if it matches, forget about all others it improves upon
        */
+
+      // the pt for views can have embedded unification type variables or BoundedWildcardTypes which
+      // can't be solved for. Rather than attempt to patch things up later we just skip those cases
+      // altogether.
+      lazy val wildPtIsInstantiable =
```

Comment:
Peeling of a layer of negations, we get: `lazy val skipMatchesPt = isView && wildPt.exists (...)` and `if(skipMatchesPt || matchesPt...)`